nkms/crypto/powers.py

```diff
@@ -55,7 +55,8 @@ def sign(self, *messages):
             sig_keypair = self._power_ups[SigningKeypair]
         except KeyError:
             raise NoSigningPower
-        msg_digest = sig_keypair.digest(*messages)
+        # msg_digest = sig_keypair.digest(*messages)  #  This didn't work.  I guess we don't hash things before signing / verifying?
```

Comment:
@tuxxy, @michwill: So it looks like we *don't* expect messages to have been hashed before signing / verifying (contrary to the docstrings in `sign` and `verify`).

What's the best practice here?  Shall we actually hash the message first?  Or no?